### ISSUE TITLE
Error while building for macos: expressions are not allowed at the top level

### ISSUE THREAD

**Issue by szediwy** [kineapps/flutter_archive#43]
```
[...]/.pub-cache/hosted/pub.dartlang.org/flutter_archive-4.1.0/macos/Classes/SwiftFlutterArchivePlugin.swift:1:1: error: expressions are not allowed at the top level
../../ios/Classes/SwiftFlutterArchivePlugin.swift
^
```

I guess there is some problem with symlinks like described in #26

<img width="1180" alt="Screenshot 2021-11-26 at 15 30 57" src="https://user-images.githubusercontent.com/4198982/143596017-af550345-6fdd-4bd0-aebb-bebc30c16cd9.png">
solves the issue

**Comment by a-v-ebrahimi** [comment]
Same as here, I could run on macOS previously, but currently I can't!

**Comment by a-v-ebrahimi** [comment]
As a temporary work around you can copy two classes from iOS folder!

**Comment by lvsecoto** [comment]
I copy `ios/Classes/SwiftFlutterArchivePlugin.swift` to `macos/Classes/SwiftFlutterArchivePlugin.swift`, and problem fixed. But it's too hacked

**Comment by kinex** [comment]
Fixed in 4.1.1
